spec/factories/users.rb

```diff
@@ -4,5 +4,7 @@
     sequence(:uid) { |i| "uid#{i}" }
     sequence(:nickname) { |i| "nickname#{i}" }
     sequence(:image_url) { |i| "http://example.com/image#{i}.jpg" }
+    sequence(:email) { |i| "test#{i}@test.com" }
+    sequence(:password) { |i| "password" }
```

Comment:
__[rubocop]__
* Lint/UnusedBlockArgument: Unused block argument - `i`. You can omit the argument if you don't care about it. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)